assets/base/i18n/en.json

```diff
@@ -1703,6 +1703,9 @@
   "mobile.post.delete_question": "Are you sure you want to delete this post?",
   "mobile.post.delete_title": "Delete Post",
   "mobile.post.retry": "Refresh",
+  "mobile.post.failed_delete": "Delete Message",
+  "mobile.post.failed_retry": "Try Again",
+  "mobile.post.failed_title": "Unable to send your message",
```

Comment:
Cancel is actually supposed to be built into the options_modal scene. I went ahead and included the fix for that to use the localization instead of just the string.